src/pages/STG_PanelOppBatch.page

```diff
@@ -1,18 +1,19 @@
 <apex:page controller="STG_PanelOppBatch_CTRL" >
-    <c:STG_PageHeader sectionLabel="{!$Label.stgNavDonations}" pageLabel="{!$Label.stgNavRollupBatch}" />
-	<p>{!$Label.stgHelpBatchOppRollup}</p>
-
-    <apex:form id="form" styleClass="form-horizontal">   
-        <apex:pageMessages id="pageMessages"/>
-        <apex:pageBlock id="pbRelBatch" mode="maindetail">
-            <div class="button-block">
-                <apex:commandButton value="{!$Label.stgBtnRunBatch}" status="statusLoad" action="{!runBatch}" disabled="{!isRunningBatch}" immediate="true" rerender="pbRelBatch" styleClass="btn-sm btn-primary" />
-            </div>
-              
-            <apex:outputPanel rendered="{!isRunningBatch}">
-                <c:UTIL_JobProgress strBatchComponentLabel="{!$Label.stgLabelOppBatchProgress}" cNumberOfJobs="4" bRerenderOnComplete="true" />
-            </apex:outputPanel>
-
-        </apex:pageBlock>
+    <apex:form id="form" styleClass="slds-m-around--x-large">
+        <c:STG_PageHeader sectionLabel="{!$Label.stgNavBulkProcesses}" pageLabel="{!$Label.stgNavRollupBatch}" />
+        <div class="slds-text-body--small slds-m-around--medium">{!$Label.stgHelpBatchOppRollup}</div>
+        <c:UTIL_PageMessages />
+        <div class="slds-grid slds-grid--align-center slds-grid--vertical-align-center slds-p-around--large">
+            <apex:commandButton value="{!$Label.stgBtnRunBatch}" status="statusLoad" action="{!runBatch}" disabled="{!isRunningBatch}" immediate="true" rerender="form" styleClass="slds-button slds-button--small slds-button--neutral" />
+        </div>
+        <apex:outputPanel rendered="{!isRunningBatch}">
+            <c:UTIL_JobProgressLightning id="jobProgress"
```

Comment:
So something is wrong with the job progress component.  Frequently when I tried running settings batches, the page would reload (as expected) and show the outline of the widget, but it would be stuck empty with the animated cursor, until it finally (minutes later) displayed at 100%.  This was not consistent, and did not happen every time.  But I did reproduce it in most of the batch panels.  It may be we aren't getting the first call that we expect out of the batch for quite some time, but then we should change our widget so it isn't completely empty until it gets it first 5%.  I reproduced this both under LEX and Classic.  Here is an example of one that has been running now for 5 minutes!
![image](https://cloud.githubusercontent.com/assets/908119/13762632/67930b58-e9fd-11e5-844e-13972c4e348b.png)